RELEASING.md

```diff
@@ -0,0 +1,292 @@
+How to Create a Release of GRPC Java (for Maintainers Only)
+===============================================================
+
+Build Environments
+------------------
+We deploy GRPC to Maven Central under the following systems:
+- Ubuntu 14.04 with Docker 1.6.1 that runs CentOS 6.6
+- Windows 7 64-bit with MSYS2 with mingw32 and mingw64
+- Mac OS X 10.9.5
+
+Other systems may also work, but we haven't verified them.
+
+Prerequisites
+-------------
+
+### Setup OSSRH and Signing
+
+If you haven't deployed artifacts to Maven Central before, you need to setup
+your OSSRH (OSS Repository Hosting) account and signing keys.
+- Follow the instructions on [this
+  page](http://central.sonatype.org/pages/ossrh-guide.html) to set up an
+  account with OSSRH.
+- (For release deployment only) Install GnuPG and [generate your key
+  pair](https://www.gnupg.org/documentation/howtos.html). You'll also
+  need to [publish your public key](https://www.gnupg.org/gph/en/manual.html#AEN464)
+  to make it visible to the Sonatype servers
+  (e.g. ``gpg --keyserver pgp.mit.edu --send-key <key ID>``).
```

Comment:
I think I'll make the change throughout ... I wondered why this was being done ... so I just tried to be consistent :)